core-bundle/contao/dca/tl_image_size.php

```diff
@@ -417,6 +438,23 @@ public function getFormats(DataContainer $dc=null)
 		return $options;
 	}
 
+	/**
+	 * Return the image metadata options
+	 *
+	 * @param DataContainer $dc
+	 *
+	 * @return array
+	 */
+	public function getMetadata(DataContainer $dc=null)
+	{
+		foreach ((new ResizeOptions())->getPreserveCopyrightMetadata() as $key => $value)
+		{
+			$options[serialize(array($key => $value))] = strtoupper($key) . ' (' . implode(', ', iterator_to_array(new RecursiveIteratorIterator(new RecursiveArrayIterator($value)))) . ')';
```

Comment:
Are we sure that we want to store `serialize(array($key => $value))` here? I don't know of a better alternative atm but just wondering if that's the best we can do?

Reply:
Also, I'd actually love to see more listeners for the options rather than our funny `tl_*` classes.

Reply:
> I don't know of a better alternative atm but just wondering if that's the best we can do?

I think this should be good enough. Creating a seperate child table/DCA seems overkill to me.